torch/csrc/api/src/nn/modules/loss.cpp

```diff
@@ -48,6 +48,27 @@ Tensor CosineEmbeddingLossImpl::forward(
     const Tensor& target) {
   return F::cosine_embedding_loss(input1, input2, target, options);
 }
+// ============================================================================
+
+MultiLabelSoftMarginLossImpl::MultiLabelSoftMarginLossImpl(
+    const torch::nn::MultiLabelSoftMarginLossOptions& options_)
```

Comment:
clang-tidy seems to complain about this line, and we can do the following to get around it:
```suggestion
    const torch::nn::MultiLabelSoftMarginLossOptions& options_) // NOLINT(modernize-pass-by-value)
```